.gitignore

```diff
@@ -16,3 +16,4 @@ pyvenv.cfg
 dist/
 /.coverage
 /.coverage.*
+.envrc
```

Comment:
I think, this one is unrelated.

I don't mind, but this feels like something that should go in your global gitignore file. :)

Here is how mine is set up:

```
touch ~/.gitignore_global
git config --global core.excludesfile ~/.gitignore_global
```

```
>cat ~/.gitignore_global
.DS_Store
.direnv
.envrc
.env
.idea
__pycache__/
```